config/component_metadata.yaml

```diff
@@ -1,4 +1,4 @@
 releases:
-  - name: Kubeflow Pipelines
+  - name: Red Hat build of Kubeflow Pipelines
```

Comment:
So repoUrl is pointed to KFP, so would this make the metadata misleading?

Reply:
should be fine IMO, since we specifically mention it as a build of KFP.